### DISCUSSION TITLE
.PES file not showing on brother SE600

### DISCUSSION THREAD

**Discussion by awinborne415** [inkstitch/inkstitch#1699] [Q&A] [answered]
Hello,

This is my first time using inkstitch. I have done the params and applied it to the file and then exported as a .pes and I tried exporting as a .zip with multiple files but neither will show up on my Brother SE600 when I plug in the USB. Any idea s to why this is? I saw an old thread for a different version having issues with it however, I have the newest version of inkstitch.

**Comment by tyrosinase** [comment]
The "fun" thing about hoop sizes is that they're all wrong. An SE600 is called a "4x4" machine but it's really 3.976" or some such... or 100mm. Some machines are even as small as 98mm. And most Brothers will not display a file that's outside its size range, so that's the first thing to look for.

Shrink your design down to 98x98 and you should see the PES (but not the ZIP). If that works, try bumping it back up to 100x100.

4x4 is generally the only size that's smaller than its metric equivalent, except I've discovered that some 6x10 machines are 160mm wide... but some are only 150mm. https://silverseams.com/embroidery-patterns/basting-boxes/index.html

**Reply by awinborne415** [reply]
Thank for that info. Unfortunately that didnt fix the problem. The file still doesn't show up when I plug into the machine.

**Reply by tyrosinase** [reply]
Hmmmmm. If you open the PES with the computer does it show the stitches correctly?

And do other PES files show up on that stick? I think SE600's are finicky about the formatting of their USB sticks. If not, you probably want to format it to 2GB/FAT just to be sure. I use Linux but https://www.instructables.com/Format-USB-Flash-Drive-to-FATFAT16-not-FAT32/ should explain how do it in Windows.

**Reply by buzzmandt** [reply]
When you do a troubleshoot does it come back with all objects are valid? 
Or when you do a simulated preview does it stitch out ok?
You are certain the pattern design is not over size (less then a 4"x4" area)?
Try saving as a .dst file, which brother machines can also read just fine, but is a bit more "bullet proof" of an extension.

**Reply by awinborne415** [reply]
It does show the stitches fine when I open it. The USB formatting is FAT32 which is what the brother website says it should be. 

When troubleshooting it says all objects are valid. The size is 98x50 mm. The .dst file doesn't show up either.

**Reply by tyrosinase** [reply]
Are you able to put non-Ink/Stitch PES files on that USB?

**Reply by awinborne415** [reply]
I just tried and no it wont show a none inkstitch one either. I have tried multiple different USB and its still the same. Which leaves me to think something is wrong with the machine.

**Reply by awinborne415** [reply]
I will say it does show the folders that are on the USB but it will not show the .pes files for selecting.

**Reply by awinborne415** [reply]
Just to inform as well I check to make sure the USB is compatible with Brother and it shows on their list of compatible devices

**Reply by awinborne415** [reply]
Ok, I figured out the problem. I'm just an idiot you have to have the embroidery deck attached for it to read embroidery files. So not problem with inkstitch just brother and specifics.

**Comment by awinborne415** [comment, marked as answer]
For future reference, make sure you have the embroidery deck attached, your item is 100mm or less if a 4"x4", and make sure there are less than 30,000 stitches.

**Reply by tyrosinase** [reply]
Oh I had completely forgotten about the stitch count limit! (And didn't even think about the deck issue; mine's not a combo... though the deck *does* come off.)
